templates/Segment_Image_Data.ipynb

```diff
@@ -17,10 +17,9 @@
    "source": [
```

Comment:
<p>I think <code>fovs_input</code>  and <code>input_data_xr</code>  are unneeded now that everything's in the same notebook.  In particualr, <code>data_xr</code>  should contain the same information as <code>input_data_xr</code>.   You can double check this by printing them both out, but at worst, they're just transposed.</p><p> </p><p>Also, I think the line <code>chan_list = overlay_channels[0]</code>  is unneeded</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/193/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>